test/e2e/framework/kubelet_stats.go

```diff
@@ -58,7 +58,7 @@ type KubeletLatencyMetrics []KubeletLatencyMetric
 
 func (a KubeletLatencyMetrics) Len() int           { return len(a) }
 func (a KubeletLatencyMetrics) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
-func (a KubeletLatencyMetrics) Less(i, j int) bool { return a[i].Latency > a[j].Latency }
```

Comment:
this is used in HighLatencyKubeletOperations, which is looking for the highest latency values.  The original implementation is also correct here